src/content/docs/infrastructure/microsoft-azure-integrations/get-started/introduction-azure-monitoring-integrations.mdx

```diff
@@ -14,29 +14,10 @@ redirects:
   - /docs/infrastructure/microsoft-azure-integrations
 ---
 
-Our [Microsoft Azure](https://azure.microsoft.com/) integrations allow you to monitor and report data about your Azure services to New Relic, providing a comprehensive view of your entire architecture in one place. 
+New Relic has several offerings for monitoring your [Microsoft Azure](https://azure.microsoft.com) applications and services. Our Azure offerings include: 
 
-<Callout variant="tip">
-This doc covers our Azure monitoring integrations. We also offer integrations to [deploy New Relic within the Azure Marketplace](/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native), and [support for Azure with our .NET agent](/docs/apm/agents/net-agent/azure-installation/install-net-agent-azure-web-apps/). 
-</Callout>
+* New Relic in Azure: With our [Azure Native New Relic Service](/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native), you can deploy New Relic from the Azure Marketplace. 
+* Our Azure infrastructure integrations: We have [dozens of Azure integrations](/docs/infrastructure/microsoft-azure-integrations/get-started/activate-azure-integrations) to monitor the Azure services you use. 
```

Comment:
In this bullet I'd want to emphasize how trivial it is to start monitoring your Azure services in New Relic this way.